BraveRewardsUI/Wallet/WalletViewController.swift

```diff
@@ -96,6 +98,11 @@ class WalletViewController: UIViewController, RewardsSummaryProtocol {
       navigationController?.setNavigationBarHidden(true, animated: animated)
     }
     reloadUIState()
+    
+    // Not sure why this works
+    DispatchQueue.main.asyncAfter(deadline: .now()) {
+      self.configureSize()
```

Comment:
Doing this without `asyncAfter` shows no change. Maybe because of the animations in PopoverController's `preferredContentSizeDidChange`